### ISSUE TITLE
Also compresses images

### ISSUE THREAD

**Issue by madc** [dactivo/Bolt.cm-HTML-minifier#2]
It seems, the extension also compresses images that are processed by Bolt.

In my case, I had to extend `Extension.php` a bit:

```
[...]
if($this->getExtension($request->getPathInfo())!=".css" && 
$this->getExtension($request->getPathInfo())!=".js"  && 
$this->getExtension($request->getPathInfo())!=".jpg")
[...]
```

This is just a quick fix for `.jpg`, but I#m sure there is a more generic solution to this issue.

**Comment by dactivo** [comment]
Yes I saw that, and as a quick fix, I added the extensions css and js, because the filter $this->app->after receives any file / request. 

I have found a better solution, I think:
if($this->getExtension($request->getPathInfo())=="")

Because html is what the plugin compresses, and no extension response to the page request. I don't know any other method to verify this in silex / bolt.cm.

What do you think about?

**Comment by madc** [comment]
But wouldn't this fail, if you have routes suffixed with '.html'?

**Comment by dactivo** [comment]
Bolt.cm does not add this suffix, no?

**Comment by madc** [comment]
Not by default, no. But is should be possible to archive such a link by modifying the routing setup.
